mce-fbdev.h

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file mce-fbdev.h
+ * Frame buffer device handling code for the Mode Control Entity
+ * <p>
+ * Copyright 2015 Jolla Ltd.
+ * <p>
+ * @author Simo Piiroinen <simo.piiroinen@jollamobile.com>
+ *
+ * mce is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation.
+ *
+ * mce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with mce.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef MCE_FBDEV_H_
+# define MCE_FBDEV_H_
+
+# include <stdbool.h>
+
+# ifdef __cplusplus
+extern "C" {
+# elif 0
+} /* fool JED indentation ... */
```

Comment:
What about [GNU Indent](https://www.gnu.org/software/indent/)? I'm sure it can be configured to do the right indentation style for mce (and would allow contributors to just run indent with the config file someone provides for mce before submitting a patch). To my knowledge, mce doesn't contain any z80 code, so that point is - for the mce codebase itself - moot ;)